core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -232,6 +263,29 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
     }
   }
 
+  calculatePanelHeight(): void {
+    setTimeout(()=>{
```

Comment:
Nit: Prefer whitespace around `=>` if linter agrees.
```suggestion
    setTimeout(() => {
```

Reply:
Fixed